internal/charmstore/search.go

```diff
@@ -610,7 +611,7 @@ func nameFilter(value string) elasticsearch.Filter {
 // owner taken from the URL.
 func ownerFilter(value string) elasticsearch.Filter {
 	if value == "" {
-		return elasticsearch.ExistsFilter("PromulgatedURL")
+		return promulgatedFilter("1")
```

Comment:
Are there any other boolean filters?
I wonder if searching for promulgated=yes should really
search for non-promulgated charms?